manifests/m/Mozilla/Firefox/Beta/100.0/Mozilla.Firefox.Beta.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mozilla.Firefox.Beta
+PackageVersion: "100.0"
+PackageLocale: en-US
+Publisher: Mozilla Corporation
```

```suggestion
Publisher: Mozilla
```
This must match Control Panel.